EIPS/eip-5131.md

```diff
@@ -0,0 +1,238 @@
+---
+eip: 5131
+title: ENS Subdomain Authentication
+description: Use ENS to link hot wallets as approved signers for a root ENS address.
+author: Wilkins Chung (@wwhchung)
+discussions-to: https://ethereum-magicians.org/t/eip-603-ens-authentication-link/9458
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-06-03
+requires: 137
+--- 
+
+## Abstract
+Proving the ownership of an asset to a third party application in the Ethereum ecosystem is a common occurrence. Users frequently sign payloads of data presented to them via these applications in order to prove ownership. The process of proving ownership of an address is commonly used to authenticate users before allowing them to perform some operation.
+
+Examples:
+ - In order for you to edit your profile on OpenSea, you must sign a message with your wallet address.
+ - In order to access NFT gated content, you must sign a message with the wallet containing the NFT.
+ - In order to claim an airdrop, you must interact with the smart contract with the qualifying wallet address.
+ - In order to prove ownership of an NFT, you must sign a payload with the address that owns that NFT.
+
+Unfortunately, we've seen many cases where users have accidentally signed a malicious payload. The result is almost always a significant loss of assets associated with the signing address.
+
+In addition to this, many users keep significant portions of their assets in 'cold storage'. With the increased security from 'cold storage' solutions, we usually see decreased accessibility because users naturally increase the barriers required to access these wallets.
+
+This EIP proposes a solution which uses the Ethereum Name Service Specification (EIP-137) as a way to link one or more signing wallets (presumably less secure) to authenticate a main wallet.
```

Comment:
```suggestion
This EIP proposes a solution which uses the Ethereum Name Service Specification (EIP-137) as a way to link one or more signing wallets (presumably less secure) to authenticate a main wallet for reversible operations.
```

I would specify reversible because "authentication" could mean e.g. signing transactions on behalf of the cold wallet, and you wouldn't want a hot wallet to be able to do that for obvious reasons. Specifying reversible means that it's clear that the changes made should be able to be undone.